File: app/src/test/java/igrek/songbook/dagger/BaseDaggerTestTest.java
package igrek.songbook.dagger;


import org.junit.Test;

import igrek.songbook.dagger.base.BaseDaggerTest;

public class BaseDaggerTestTest extends BaseDaggerTest {
	
	@Test
	public void testActivityInjection() {
		System.out.println("injected activity: " + activity.toString());
	}
	
	@Test
	public void testLoggerMock() {
		logger.info("Hello dupa");
	}
	
}

File: app/src/test/java/igrek/songbook/dagger/BaseDaggerTestTest.kt
package igrek.songbook.dagger


import igrek.songbook.dagger.base.BaseDaggerTest
import org.junit.Test

class BaseDaggerTestTest : BaseDaggerTest() {

    @Test
    fun testActivityInjection() {
        println("injected activity: $activity")
    }

    @Test
    fun testLoggerMock() {
        logger.info("Hello dupa")
    }

}
